ValkeyFullTextSearch.md

```diff
@@ -0,0 +1,392 @@
+│                                    │                             │            │
+│                           ┌────────┴───────┐                     │            │
+│  ┌───────────────┐        │ Result         │◀────────────────────┘            │
+│  │ Client        │◀───────│ Formatting     │                                  │
+│  │ Applications  │        │                │                                  │
+│  └───────────────┘        └────────────────┘                                  │
+│                                                                               │
+│                                                                               │
+│  ┌───────────────┐        ┌────────────────┐         ┌────────────────────┐   │
+│  │ Client        │        │ Command        │         │ Text Index         │   │
+│  │ Applications  │───────▶│ Processing     │────────▶│ Builder            │   │
+│  └───────────────┘        │ (FT.CREATE)    │         └──────────┬─────────┘   │
+│                           └────────────────┘                     │            │
+│                                                                  │            │
+│                                                                  ▼            │
+│                                                       ┌────────────────────┐  │
+│                                                       │ Tokenization       │  │
+│                                                       │ Pipeline           │  │
+│                                                       │                    │  │
+│                                                       │ 1. Split on punct. │  │
+│                                                       │ 2. Case conversion │  │
+│                                                       │ 3. Stop word filter│  │
+│                                                       │ 4. Stemming        │  │
+│                                                       └──────────┬─────────┘  │
+│                                                                  │            │
+│                                                                  ▼            │
+│                                                       ┌────────────────────┐  │
+│                                                       │ Term Storage in    │  │
+│                                                       │ Prefix/Suffix Trees│  │
+│                                                       └────────────────────┘  │
+│                                                                               │
+└───────────────────────────────────────────────────────────────────────────────┘
+
+```
+## Specification
+
+Each ```TEXT``` field has the a set of configurables some control the process to convert a string into a vector of terms, others control the contents of the generated index.
+
+| Metadata          | Type              | Meaning                                                                    | Default Value |
```

Comment:
There is some slight inconsistency between this table and the later values, just make sure they're consistent.